package.json

```diff
@@ -79,6 +80,11 @@
 				"title": "Execute Command"
 			},
 			{
+				"category": "MongoDB",
+				"command": "cosmosDB.launchMongoShell",
+				"title": "Mongo Shell"
```

Comment:
This shows up in the command palette as `MongoDB: Mongo Shell`

What if we changed the title to "Launch Shell" so that we don't see "Mongo" twice